packages/main/src/Input.js

```diff
@@ -754,13 +760,16 @@ class Input extends UI5Element {
 		}
 	}
 
-	_handleChange(event) {
-		if (!this._changeFired) {
+	_handleNativeInputChange() {
+		clearTimeout(this._nativeChangeDebounce);
```

Comment:
can we document why we need that timeout and that debounce? its quite unclear to me

Reply:
maybe to ensure that `this.value` has a correct value?

Reply:
Hmmmm, strange! I think I have put on some comments for that. The thing is that there are some cases where the native change event is fired **before** the value actually changes and the **_value_** property is the old one. This is the reason for this "dobeounce"